Src/DeUrgenta.RecurringJobs/Jobs/ExpiredCertificationJob.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using DeUrgenta.Domain;
+using DeUrgenta.RecurringJobs.Jobs.Config;
+using DeUrgenta.RecurringJobs.Services;
+using Microsoft.Extensions.Options;
+
+namespace DeUrgenta.RecurringJobs.Jobs
+{
+    public class ExpiredCertificationJob : IExpiredCertificationJob
+    {
+        private readonly DeUrgentaContext _context;
+        private readonly INotificationService _notificationService;
+        private readonly ExpiredCertificationJobConfig _config;
+
+        public ExpiredCertificationJob(DeUrgentaContext context, INotificationService notificationService, IOptionsMonitor<ExpiredCertificationJobConfig> config)
+        {
+            _context = context;
+            _notificationService = notificationService;
+            _config = config.CurrentValue;
+        }
+
+        public async Task RunAsync()
+        {
+            var usersWithExpiredCertifications = _context.Certifications
```

Comment:
we should make sure we will not spam users.
I think we should find a mechanism to exclude previously notified users. 

We should send 2 emails:
* one x days before the certification expiration
* one when certification is expiried
A simple solution is to keep track of notified users about their certifications. Add a database context with a couple of tables but make sure they will have a "jobs" schema so they will not interfere with other tables.
Please implement a mechanism for cleaning these exclusion tables.

Reply:
Done, and added #107 to add the implementation for the Email sender as discussed.